civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -16,6 +16,70 @@ $row-margin-bottom: 15px;
   margin-bottom: 20px;
 }
 
+.chr_leave-request-modal__form {
+
+  .ui-select-container {
+    .text-muted {
+      color: $brand-primary;
+      font-size: $crm-font-size-large;
+    }
+
+    &:not(.ui-select-multiple) {
+      .glyphicon-remove {
+        top: 5px;
+      }
+
+      .ui-select-match-text span {
+        font-size: $crm-font-size-large;
+      }
+    }
+
+    input[type="search"] {
+      color: $brand-primary;
+      font-size: $crm-font-size-large;
+      border: none !important;
+    }
+  }
+
+  .form-control::-webkit-input-placeholder {
+    /* Chrome/Opera/Safari */
+    color: $brand-primary;
+  }
+
+  .form-control::-moz-placeholder {
+    /* Firefox 19+ */
+    color: $brand-primary;
+  }
+
+  .form-control:-ms-input-placeholder {
+    /* IE 10+ */
+    color: $brand-primary;
+  }
+
+  .form-control:-moz-placeholder {
+    /* Firefox 18- */
+    color: $brand-primary;
+  }
+
+  .ui-select-placeholder {
+    color: $brand-primary;
+    font-size: $crm-font-size-large;
+  }
+
+  .ui-select-bootstrap > .ui-select-match > .btn {
```

Comment:
Just `.ui-select-match > .btn` is fine. `.ui-select-bootstrap` and `.ui-select-container` is on same level so no need to mention.